app/src/main/res/values/strings.xml

```diff
@@ -1,22 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <string name="app_name" translatable="false">LibreSudoku</string>
-    <string name="nav_bar_statistics">Statistics</string>
+    <string name="nav_bar_statistics">Stats</string>
     <string name="nav_bar_home">Home</string>
     <string name="nav_bar_more">More</string>
     <string name="new_game">New game</string>
     <string name="dialog_new_game">Game in progress</string>
-    <string name="dialog_new_game_text">You have an unfinished game, do you really want to start a new one?</string>
+    <string name="dialog_new_game_text">Postpone this game and start a new one?</string>
     <string name="dialog_new_game_positive">Start new</string>
     <string name="action_cancel">Cancel</string>
     <string name="action_confirm">Yes</string>
-    <string name="action_play">Play</string>
+    <string name="action_play">Start</string>
```

Comment:
In my head "Play" makes sense if the board is shown.
Then the action is to play the presented game.
"Play" without it could be a section of a menu.
I wasn't sure what "Play" would do at first. "Continue" isn't there for context right away.
It was hard for me to figure out where the "main" screen is because of this.